e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/okhttp3/WmHttpOkHttp3EMTest.kt

```diff
@@ -51,13 +51,13 @@ class WmHttpOkHttp3EMTest : SpringTestBase() {
                 assertTrue(solution.individuals.size >= 1)
                 assertHasAtLeastOne(solution, HttpVerb.GET, 200, "/api/wm/socketconnect/string", "OK")
                 assertHasAtLeastOne(solution, HttpVerb.GET, 200, "/api/wm/socketconnect/string", "Hello There")
-                assertHasAtLeastOne(solution, HttpVerb.GET, 400, "/api/wm/socketconnect/string", null)
-                assertHasAtLeastOne(solution, HttpVerb.GET, 500, "/api/wm/socketconnect/string", null)
+                assertHasAtLeastOne(solution, HttpVerb.GET, 400, "/api/wm/socketconnect/string", "")
+                assertHasAtLeastOne(solution, HttpVerb.GET, 500, "/api/wm/socketconnect/string", "")
 
                 assertHasAtLeastOne(solution, HttpVerb.GET, 200, "/api/wm/socketconnect/object", "OK")
             },
-            3
+            10
```

Comment:
hmmm... unsure about this. ll need to discuss

Reply:
Forget to clean it.